interestingness/envVars.py

```diff
@@ -16,7 +18,7 @@
 ENV_PATH_SEPARATOR = ';' if os.name == 'nt' else ':'
 
 
-def envWithPath(path, runningEnv=os.environ):
+def envWithPath(path, runningEnv=os.environ):  # pylint: disable=dangerous-default-value
```

Comment:
This is again better fixed than disabled.
Make the default `None` and add a line in the function like: `runningEnv = runningEnv or os.environ`